common/changes/@itwin/core-backend/fontMapCase_2024-11-14-21-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "IModel.test.ts now validates that FontMap.getFont uses case insensitive matching for font names",
```

Comment:
```suggestion
      "comment": "",
```

change logs are for end consumers, and they wont care about tests like this

Reply:
Thanks, suggestion committed